biblio.json

```diff
@@ -26,5 +26,15 @@
         ],
         "href": "https://solidos.solidcommunity.net/",
         "title": "SolidOS"
+    },
+    "WebID": {
+        "authors": [
+            "Andrei Sambra",
+            "Henry Story",
+            "Tim Berners-Lee"
+        ],
+        "href": "https://www.w3.org/2005/Incubator/webid/spec/identity/",
+        "title": "WebID 1.0",
+        "publisher": "WebID Incubator Group"
```

Comment:
...except that the WebID XG never published this, with or without a version declaration. It would be better to title it as what it was, `"WebID 1.0 Editor's Draft"` (which must be read and understood as `"Editor's Draft of WebID 1.0"`) ... and of course, the document now served from that location says "[Draft Community Group Report](https://www.w3.org/standards/types#reports) 05 March 2014", which is similarly but differently inaccurate and contrary to the "publisher" identified here.

Reply:
In case we can't quickly resolve it in this PR I created a dedicated issue for it https://github.com/solid/security-considerations/issues/10